### PR TITLE
fix: Allow non-string literals

### PR DESCRIPTION
**Pull request by eps1lon**
Literals are now sorted by:
- numbers last, ascending
- anything else by their stringified value using localeCompare

Without this change `typescript-to-proptypes#inject` crashes.

### CODE REVIEW

File: src/types/props/literal.ts [hunk 1]
```diff
@@ -3,11 +3,11 @@ import { Node } from '../nodes/baseNodes';
 const typeString = 'LiteralNode';
 
 export interface LiteralNode extends Node {
-	value: any;
+	value: unknown;
```
Comment (merceyz) [inline src/types/props/literal.ts:6]: Why was this change needed?

Comment (eps1lon) [reply src/types/props/literal.ts:6]: To add type safety. This could've been used earlier to catch this bug.

Comment (merceyz) [reply src/types/props/literal.ts:6]: Ah, right, of course.

**Comment by merceyz** [comment]
Thank you! :+1:
